### PR TITLE
Add recipe for xproperty

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Merge!!!!

**Comment by SylvainCorlay** [comment]
ping! @conda-forge/core this is ready to go and pretty straightforward.

File: recipes/xproperty/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D BUILD_TESTS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/xproperty/bld.bat]: Is there a `make test`/`check` or `ctest` we can run here?

File: recipes/xproperty/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DBUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
```
Comment (ocefpaf) [inline recipes/xproperty/build.sh]: Same as above. (BTW, it is usually more readable to break that command into multiple lines.)

File: recipes/xproperty/bld.bat (hunk 1 above)
Comment (SylvainCorlay) [inline recipes/xproperty/bld.bat]: They require a C++14 compiler...

**Comment by SylvainCorlay** [comment]
Thanks

File: recipes/xproperty/bld.bat (hunk 1 above)
Comment (ocefpaf) [inline recipes/xproperty/bld.bat]: If you can add a comment or open an issue in the feedstock about that later so we can track this. A C++14 compiler may be available soon :wink:

Comment (isuruf) [inline recipes/xproperty/bld.bat]: You can use `clangdev` package which gives you clang 4.0.0 with `libstdc++` as default standard library.

**Pull request merged**